### ISSUE TITLE
Various words incorrectly recognized as nouns when starting with uppercase

### ISSUE THREAD

**Issue by Arcyvilk** [spencermountain/compromise#647]
I'm trying to extract all names from the text using the ``ProperNoun`` tag but I'm consistently getting words which should not be classified as one.

Those include words like "Strange", "Irresponsibly", "As", "Was", "Light" etc.
Examples of those from ``.debug()``:

![image](https://user-images.githubusercontent.com/12732348/70869840-1d42fa80-1f8d-11ea-8775-346577d7443a.png)

It can be solved by lowercasing the analysed text but it also causes real names to not be recognized as ``ProperNoun`` anymore.

**Comment by spencermountain** [comment]
hey thanks @Arcyvilk , can you make sure you're on v12?
it seems to work as expected here:
https://runkit.com/spencermountain/5dfa5d8ebf9a60001a676fde

cheers

**Comment by Arcyvilk** [comment]
I'm using version 12.2.1.

I did some more tests and it seems it might be related to #652 . When the word "Pretty" is the first word of the entire text, it's properly recognized as ``Adjective``. If it's still the first word in the sentence, but its sentence is not the first sentence of the entire text, it's assigned tags `Noun` and `ProperNoun`.

I reproduced it here: https://runkit.com/arcyvilk/5dfaabc1ca4a2f001a350212
